stages/org.osbuild.vagrant

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+"""
+Creates the required files for a Vagrant box, intended to be used in with the
+org.osbuild.tar stage to tar up the metadata, Vagrantfile and VM image.
+
+TODO: vagrant-virtualbox support
```

Comment:
I would get rid of this and instead file an issue on osbuild if you wanna keep track of it.